test/e2e/resources/pipeline-maven.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       type: string
 
   tasks:
-    - name: git
+    - name: git-clone
```

Comment:
nit (for the future): we could have kept that name as `git` as it doesn't need to be the same as the "task name" itself 😛

Reply:
that's true, but it made more sense that's all